### ISSUE TITLE
Plugin needs to be updated in order to be compatible with GoLand 2021.2 EAP?

### ISSUE THREAD

**Issue by mojotx** [JetBrains/idea-gitignore#709]
### Prerequisites

* [X] Plugin is in the latest version
* [X] Issue was not reported yet
* [ ] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Description

This plugin needs to be updated in order to be compatible with GoLand 2021.2 EAP. 

### Steps to Reproduce

1. Install GoLand 2021.1 with .ignore plugin
2. Update to GoLand 2021.2 EAP
3. See error message

### Versions

**Plugin:** 

```
.ignore
4.1.0 JetBrains
Incompatible: requires IDE build 211.* or older
```

**IDE:** 

```
GoLand 2021.2 EAP
Build #GO-212.3116.33, built on May 26, 2021
GoLand EAP User
Expiration date: June 25, 2021
Runtime version: 11.0.11+9-b1460.1 x86_64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
macOS 11.4
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 8
Registry: ide.completion.variant.limit=500, suggest.all.run.configurations.from.context=true
Non-Bundled Plugins: IdeaVIM (0.67), bundled-goland-help (212.3116.33), com.4lex4.intellij.solarized (2.3.0), com.zt.json-editor (2.2), com.intellij.kubernetes (212.3116.29), org.toml.lang (0.2.147.3871-211), com.zxy.ijplugin.wechat-miniprogram (3.5.4), idea.plugin.protoeditor (212.3116.45)
```

**OS:** 

```
macOS 11.4 "Big Sur"
```

### Additional Information

I am not a Java developer, but I believe the gradle.properties file just needs to be updated, and bump the values of `pluginSinceBuild` and `pluginUntilBuild` from 211 to 212, correct? 

<img width="595" alt="Screen Shot 2021-06-01 at 12 06 26 PM" src="https://user-images.githubusercontent.com/5694899/120364882-639c2e80-c2d3-11eb-9981-508c7463d77a.png">

**Comment by ForsakenHarmony** [comment]
It's a bit ironic that they used this as an example for [how to update plugins](https://blog.jetbrains.com/platform/2021/01/revamping-plugins-ignore/) but then stop updating it quickly soon after

**Comment by mojotx** [comment]
Another error when I started this morning. The plugins are installed, but not enabled. 
<img width="1238" alt="Screen Shot 2021-06-02 at 10 02 01 AM" src="https://user-images.githubusercontent.com/5694899/120504801-0dd68d80-c38a-11eb-8d92-01c0697a83b2.png">

**Comment by hsz** [comment]
.ignore plugin is now available for 212 as v4.1.1
